static/tests/frontend/specs/commentIconsGroups.js

```diff
@@ -0,0 +1,212 @@
+  });
+
+  context('when a comment is deleted from a group of two icons', function(){
+    before(function(done){
+      apiUtils.simulateCallToDeleteComment(commentIds[INDEX_OF_FIRST_ID_OF_FIRST_GROUP+1]);
+      helper.waitFor(function(){
+        return helperFunctions.
+          getCommentIconsContainer(commentIds[INDEX_OF_FIRST_ID_OF_FIRST_GROUP]).
+          children().
+          length === 1;
+      }).done(done);
+    });
+
+    after(function(){
+      utils.undo();
+    });
+
+    it('removes the ellipsis from the remaining icon and shows icon', function(done){
+      var $icon = helperFunctions.getCommentIcon(commentIds[INDEX_OF_FIRST_ID_OF_FIRST_GROUP])
+      helper.waitFor(function(){
+        return $icon.length &&
+               !$icon.hasClass('with-ellipsis') &&
+               $icon.css('opacity') == 1;
+      }).done(done);
+    });
+
+  });
+
+  context('when a comment is deleted from a group with more than two icons', function(){
+    before(function(done){
+      apiUtils.simulateCallToDeleteComment(commentIds[INDEX_OF_FIRST_ID_OF_SECOND_GROUP]);
+      helper.waitFor(function(){
+        return helperFunctions.
+        getCommentIconsContainer(commentIds[INDEX_OF_FIRST_ID_OF_SECOND_GROUP + 1]).
+        children().
+        length === 2;
+      }).done(done);
+    });
+
+    after(function(){
+      utils.undo();
+    });
+
+    testIfCommentLineIsGrouped(INDEX_OF_FIRST_ID_OF_SECOND_GROUP + 1, SECOND_GROUP_LENGTH - 1);
+
+  });
+
+  function testIfCommentLineIsGrouped(indexOfFirstIconOfGroup, expectedLength){
+    it('groups all comment icons on that line', function(done){
+      helper.waitFor(function () {
+        var $commentLine = helperFunctions.getCommentIconsContainer(commentIds[indexOfFirstIconOfGroup]);
+        return $commentLine.children().length === expectedLength;
+      }).done(done);
+    });
+
+    it('sets the first icon as visible and adds an ellipsis to it', function (done) {
+      helper.waitFor(function(){
+        var $commentLine = helperFunctions.getCommentIconsContainer(commentIds[indexOfFirstIconOfGroup]);
+        var $firstIcon = $commentLine.children().first();
+        return $firstIcon.length &&
+               $firstIcon.hasClass('with-ellipsis') &&
+               $firstIcon.css('opacity') == 1;
+      }).done(done);
+    });
+
+    it('hides the other icons on that line', function (done) {
+      helper.waitFor(function(){
+        var $commentLine = helperFunctions.getCommentIconsContainer(commentIds[indexOfFirstIconOfGroup]);
+        var $comments = $commentLine.children();
+
+        for(var i = 1; i < $comments.length; i++){
+          var $icon = $($comments[i]);
+          if($icon.hasClass('with-ellipsis') || $icon.css('opacity') == 1 ){
+            return false;
+          }
+        }
+        return true;
+      }).done(done);
+    });
+  }
+
+});
+
+var ep_comments_page_test_helper = ep_comments_page_test_helper || {};
+
+ep_comments_page_test_helper.commentIcons = {
```

Comment:
Como `ep_comments_page_test_helper.commentIcons` já foi definido no outro arquivo de testes, precisamos renomear aqui pra evitar algum conflito quando todos os testes são executados juntos.

```suggestion
ep_comments_page_test_helper.commentIconsGroups = {
```